clients/linux/src/app.rs

```diff
@@ -196,9 +208,13 @@ impl LbApp {
             if let Some(msg) = msgopt {
                 lb.gui.intro.sync_progress(&msg)
             } else if let Err(err) = lb.gui.show_account_screen(&lb.core) {
-                lb.messenger.send_err("showing account screen", err);
+                lb.messenger.send_err_dialog("showing account screen", err);
             } else {
                 lb.messenger.send(Msg::RefreshSyncStatus);
```

Comment:
This is a network call, it should happen in it's own thread. This will lock up the UI while they're typing for someone with a slow network connection.